tests/dummy/app/components/navigation-menu.css

```diff
@@ -11,7 +11,7 @@
   white-space: nowrap;
 }
 
-.link.active-route {
+.link:global(.active) {
```

Comment:
Ember provides the `.active` class so that, when we use the `<LinkTo>` component, we can highlight the link when the current route matches the link's URL.

The `:global` keyword is needed because we use `ember-css-modules` for the demo app.